app/views/user_resources/_form.html.erb

```diff
@@ -10,29 +10,24 @@
       </ul>
     </div>
   <% end %>
-  <div class="new_resource_wrapper">
+  <div class="new_resource_wrapper gray-boxing">
     <div id="selection_field">
-      <div>
-        <%= form.label "#{t('user_resources.file')}" %>
-        <%= form.file_field :file, required: true%>
-      </div>
-      <div>
-        <%= form.label "#{t('user_resources.name')}" %>
-        <%= form.text_field :name, required: true %>
-      </div>
-      <div>
-        <%= form.label "#{t('user_resources.desc')}" %>
-        <%= form.text_area :desc, required: true %>
-      </div>
-      
-      <div>
-        <%= form.label "#{t('user_resources.tag')}" %>
-        <%= form.text_field :tag, required: true %>
-      </div>
+      <%= form.label "#{t('user_resources.file')}" %>
+      <%= form.file_field :file, required: true %>
+
+      <%= form.label "#{t('user_resources.name')}" %>
+      <%= form.text_field :name, required: true %>
+
+      <%= form.label "#{t('user_resources.desc')}" %>
+      <%= form.text_area :desc, required: true %>
+
+      <%= form.label "#{t('user_resources.tag')}" %>
+      <%= form.text_field :tags, required: true %>
     </div>
 
     <div id="createNewResource_button">
       <%= form.submit %>
+      <%= button_to "#{t('user_resources.back')}", user_resources_path, method: :get %>
```

Comment:
Wozu wird hier `method: :get` benötigt? Sollte nicht der Fall sein. Eventuell falsche Benutzung des Forms?

Reply:
leider wird es benötigt, 
button_to hat bei mir bisher immer einen post hervorgerufen. egal ob in einem form oder nicht

Reply:
Benutze hier `link_to`, das ist immer ein Get. Das mit Button kann ich mir gerade nicht anschauen, sollte aber auch ohne `method` richtig funktionieren.

Reply:
ich möchte aber hier einen button haben.
was ist denn eigentlich daran so falsch, ```method: :get``` 
zu nutzen?

Reply:
Das ist an sich nichts Falsches, es zeigt nur, dass hier Form bzw. Route-Helper nicht richtig benutzt wird. Weiter kann ich nicht helfen.

Reply:
https://apidock.com/rails/ActionView/Helpers/UrlHelper/button_to

```button_to``` wird immer ein POST Request durchführen, egal ob es in einem "form" context genutzt wird oder nicht

da ich hier einen button haben möchte, werde ich button_to nutzen, da dies eine eingebaute methode ist, und ich nicht extra den button nachbauen möchte